show/main.py

```diff
@@ -1116,10 +1117,15 @@ def rif(interface, period, verbose):
 
 # 'portchannel' subcommand ("show interfaces portchannel")
 @interfaces.command()
+@multi_asic_click_options
```

Comment:
For the config commands I add the namespace option at the command group level ( i.e if you take this example ..it will be like show interfaces -n asic0 portchannel/counters ) instead of specifying the options after the subcommand keyword viz portchannel, counters etc ..

I was looking to see if both the show and config commands follow similar approach .. @arlakshm @jleveque your thoughts ?

Reply:
@judyjoseph. IMO the CLI syntax is cleaner if we have the keywords first then followed by options/arguments.